Rubberduck.Refactorings/Rename/RenameRefactoring.cs

```diff
@@ -330,6 +334,16 @@ private void RenameVariable(RenameModel model, IRewriteSession rewriteSession)
         }
 
         private void RenameModule(RenameModel model, IRewriteSession rewriteSession)
+        {
+            //The parser needs to be suspended during the refactoring of a component because the VBE API object rename causes a separate reparse. 
+            var suspendResult = _parseManager.OnSuspendParser(this, new[] { ParserState.Ready }, () => RenameModuleInternal(model, rewriteSession));
+            if (suspendResult != SuspensionResult.Completed)
+            {
+                _logger.Warn("RenameModule failed.");
```

Comment:
This is really a nitpick but I would prefer that the message was a bit more descriptive. I would like something more like:
```
_logger.Warn("RenameModule failed because a parser suspension request could not be fulfilled. The request's result was '{suspendResult.ToString()}'.");
```